MANIFEST.in

```diff
@@ -1,9 +1,12 @@
 include *.py
 include *.txt
+include *.rst
+include .coveragerc
+include tox.ini
```

Comment:
check-manifest doesn't complain about `.travis.yml` because [people asked for that](https://github.com/mgedmin/check-manifest/issues/10#issuecomment-18551337).  There's no generic rule for ignoring dotfiles, just a [default ignore list](https://github.com/mgedmin/check-manifest/blob/58110391c941c7f6eeabc16e5dadd9def6d1e707/check_manifest.py#L509).